components/builder-web/app/BuilderApiClient.ts

```diff
@@ -19,7 +19,7 @@ import { GitHubApiClient } from "./GitHubApiClient";
 
 export class BuilderApiClient {
     private headers;
-    private urlPrefix: string = config["habitat_api_url"];
+    private urlPrefix: string = `${config["habitat_api_url"]}/v1`;
```

Comment:
Maybe `/v1` should just be in the config?

Reply:
It's on purpose because this doesn't belong in the config. This is part of the "protocol" and not part of the configuration. The client should hardcode which protocol it expects as it is _not_ actually configurable.

For example, say we had added a `v2`. Well, I could configure this client to talk to `v2` but it would have absolutely no idea how to handle it :)